app/decorators/ticket_transaction_decorator.rb

```diff
@@ -0,0 +1,32 @@
+class TicketTransactionDecorator < Draper::Decorator
+  delegate_all
+
+  def status
+    accepted ? "Inactive" : "Pending"
+  end
+
+  def receiver_email
+    booking.user.email
+  end
+
+  def payers_name
+    user = User.find(recipient_id)
+    user.first_name.to_s + user.last_name.to_s
```

Comment:
Pardon me but isn't this going to joint the user's first name and last name without spacing them out?